core/domain/feedback_services.py

```diff
@@ -115,8 +118,16 @@ def create_message(
             thread.subject = updated_subject
     thread.put()
 
+    if (user_services.is_user_registered(author_id) and len(text) > 0 and
```

Comment:
OK. We ought to, I think; if it's nontrivial to fix, though, please file a bug so that we don't forget about it?